### PR TITLE
Quickbuttons: Add source selection

### PR DESCRIPTION
**Pull request by luzip665**
See https://github.com/exaile/exaile/issues/775

### CODE REVIEW

File: plugins/quickbuttons/__init__.py [hunk 1]
```diff
@@ -177,7 +219,20 @@ def _add_button(self, setting: str) -> None:
...
+            tbs.connect("clicked", self._on_equalizer_press)
+
+        elif self._options[setting]["type"] == "audio_device_selection":
+            # @see plugins/previewdevice/previewprefs.py:65
+            from xl.player.gst.sink import get_devices
+
+            tbs = Gtk.ComboBoxText()
+            val = self._options[setting]["value"]
+            for device_name, device_id, _unused in list(get_devices()):
+                tbs.append(device_id, device_name)
```
Comment (virtuald) [inline plugins/quickbuttons/__init__.py:231]: This isn't going to account for devices being added/removed. Maybe add a refresh option (right click menu?)

Also, the combo box doesn't render properly for me (I have to scroll to see the items), but I expect that's just a GTK quirk?

Comment (luzip665) [reply plugins/quickbuttons/__init__.py:231]: You are right. I think best solution would be to connect to the combobox notify::popup-shown signal to reload the list of devices just in time.

Comment (luzip665) [reply plugins/quickbuttons/__init__.py]: The rendering seems to relate to the position of the box. If it is not on the lower end of the screen it works better..

**Comment by luzip665** [comment]
Now there are both devices listed
Both selections are updated in time

**Comment by virtuald** [comment]
I think the translation strings would be best in a separate commit. Doesn't matter to me whether it's this PR or another.

**Comment by luzip665** [comment]
I'll add the translations in a separate PR.

**Pull request closed without merging**